settings/ui/tests/run-acceptance-test.sh

```diff
@@ -40,7 +40,7 @@ clean_up() {
 
 trap clean_up SIGHUP SIGINT SIGTERM EXIT
 
-cp -r "$WEB_PATH"/tests "$testFolder"
+cp -r $(ls -d "$WEB_PATH"/tests/acceptance/* | grep -v 'node_modules') "$testFolder"
```

Comment:
Guess we should also omit all the files in `.yarn` from being copied here

Reply:
`.yarn` will not be copied with that command because it will only copy the output of `ls -d` (no `-a` in the `ls` command)